### PR TITLE
Add history to items

### PR DESCRIPTION
**Pull request by idealasgas**

### CODE REVIEW

**Review by Vadser — APPROVED** [review]
LGTM

**Review by reedzyy — APPROVED** [review]

**Comment by idealasgas** [comment]
**need code review**

**Comment by anatoliliotych** [comment]
remove this commit! `Update to the latest dev` before I start review!
This means that you've made something wrong during rebase. I need really careful work with git!

File: app/views/items/show.html.slim [hunk 1]
```diff
@@ -8,6 +8,33 @@ p
 p
   strong Category:
   = @item.category&.name
+- unless @item.created_at == @item.updated_at
```
Comment (anatoliliotych) [inline app/views/items/show.html.slim:11]: you can check version number instead of such check

File: app/views/items/show.html.slim [hunk 2]
```diff
@@ -8,6 +8,33 @@ p
...
+                th TITLE
+                th NICHE
+                th USER
+                th DATE
+            tbody
+              - @versions.each do |v|
+                tr
+                  td= v.title
+                  - if v.niche_id.nil?
+                    td somewhere
```
Comment (anatoliliotych) [inline app/views/items/show.html.slim:31]: what does it mean?

File: app/views/items/show.html.slim [hunk 3]
```diff
@@ -8,6 +8,33 @@ p
...
+                  - else
+                    td= Niche.all.find {|niche| niche.id == v.niche_id}.name
```
Comment (anatoliliotych) [inline app/views/items/show.html.slim]: why do we have this here?

File: app/views/items/show.html.slim [hunk 4]
```diff
@@ -8,6 +8,33 @@ p
...
+                  td= User.all.find {|user| user.id == v.user_id}.first_name
```
Comment (anatoliliotych) [inline app/views/items/show.html.slim]: bad approach to do this in view

**Review by anatoliliotych — CHANGES_REQUESTED** [review]
Needs demo!

File: app/views/items/show.html.slim (hunk 2 above)
Comment (idealasgas) [reply app/views/items/show.html.slim:31]: default value if niche is not set

**Pull request closed without merging**